src/bin/tikv-server.rs

```diff
@@ -602,7 +612,24 @@ fn run_raft_server(listener: TcpListener,
                           resolver,
                           snap_mgr)
         .unwrap();
-    start_server(svr, event_loop, engine);
+
+    let backup_path = {
+        let mut path = get_toml_string(config, "server.backup", Some(String::new()));
+        if path.is_empty() {
+            path = store_path.to_owned();
+            loop {
+                if path.ends_with('/') {
+                    path.pop();
+                } else {
+                    break;
+                }
+            }
+            path.push_str("_backup");
+        }
+        info!("backup path: {:?}", path);
+        path
+    };
+    start_server(svr, event_loop, engine, &backup_path);
```

Comment:
Verify `backup_path` is valid (it is valid path, a dir, and exists) before `start_server`.